ui/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@patternfly/react-core": "2.1.4",
     "@patternfly/react-table": "1.0.7",
+    "prop-types": "^15.7.2",
```

Comment:
@damienomurchu I'm also using Prop Types here: https://github.com/aerogear/mobile-security-service/blob/master/ui/src/containers/AppsTableContainer.js#L3 but I didn't need to add it as a dependency. It may already be there as part of the React or Redux dependencies?

Reply:
removed now :+1: